src/test/java/nl/dtls/fairdatapoint/repository/impl/StoreManagerImplTest.java

```diff
@@ -184,50 +190,114 @@ public void storeResource() {
     @Test
     public void deleteRource() {
         try {
-            Resource sub = f.createBNode("<http://www.dtls.nl/testSub>");
-            IRI obj = f.createIRI("<http://www.dtls.nl/testObj>");
-            Statement stmt = f.createStatement(sub, RDF.TYPE, obj);
             List<Statement> sts = new ArrayList();
-            sts.add(stmt);
+            sts.add(TESTSTMT);
             testStoreManager.storeStatements(sts);
-            testStoreManager.removeStatement(sub, RDF.TYPE, null);
+            testStoreManager.removeStatement(TESTSUB, RDF.TYPE, null);
+        } catch (StoreManagerException ex) {
+            fail("The test is not excepted to throw StoreManagerException");
+        }
+    }
+    
+    /**
+     * The test is excepted to pass
+     */
+    @DirtiesContext
+    @Test
+    public void storeStatement() {
+        try {
+            testStoreManager.storeStatements(STATEMENTS, TESTSUB);
+        } catch (StoreManagerException ex) {
+            fail("The test is not excepted to throw StoreManagerException");
+        }
+    }
+    
+    /**
+     * The test is excepted to pass
+     */
+    @DirtiesContext
+    @Test
+    public void storeStatementWithoutCtxt() {
+        try {
+            testStoreManager.storeStatements(STATEMENTS, null);
         } catch (StoreManagerException ex) {
             fail("The test is not excepted to throw StoreManagerException");
         }
     }
 
+
     /**
      * The test is excepted to retrieve return false
      *
-     * @throws RepositoryException
-     * @throws StoreManagerException
      * @throws Exception
      */
     @DirtiesContext
     @Test
-    public void checkNonExitingResource() throws RepositoryException,
-            StoreManagerException,
-            Exception {
+    public void checkNonExitingResource() throws Exception {
         String uri = "http://localhost/dummy";
-        boolean isStatementExist = testStoreManager.isStatementExist(
-                f.createIRI(uri), null, null);
+        boolean isStatementExist = testStoreManager.isStatementExist(f.createIRI(uri), null, null);
         assertFalse(isStatementExist);
     }
 
     /**
      * The test is excepted to retrieve return true
      *
-     * @throws RepositoryException
-     * @throws StoreManagerException
      * @throws Exception
      */
     @DirtiesContext
     @Test
-    public void checkExitingResource() throws RepositoryException,
-            StoreManagerException, Exception {
+    public void checkExitingResource() throws Exception {
         boolean isStatementExist = testStoreManager.isStatementExist(
                 f.createIRI(ExampleFilesUtils.TEST_SUB_URI), null, null);
         assertTrue(isStatementExist);
     }
 
+    /**
+     * The test is excepted to throw error
+     *
+     * @throws Exception
+     */
+    @DirtiesContext
+    @Test(expected = StoreManagerException.class)
+    public void deleteResouceCatchBlock() throws Exception {
```

Comment:
This test name doesn't really describe what is actually tested. (and bonus points for a typo in `Resource` 😜)

Reply:
I agree, naming classes and methods is still a problem for me my class names are getting better, but I can't say the same for methods especially test methods :-D. But please check is the new name is good enough